galaxy_importer/loaders.py

```diff
@@ -165,9 +167,31 @@ def load(self):
     def _make_name(self):
         return os.path.basename(self.rel_path)
 
-    def _lint(self):
+    def _lint_role(self, path):
         self.log.info('Linting role via ansible-lint')
-        pass
+        cmd = [
+            'ansible-lint', path,
+            '-p',
+            '-x', 'metadata',
+        ]
+        self.log.debug('CMD: ' + ' '.join(cmd))
+        proc = Popen(
+            cmd,
+            cwd=self.root,
+            encoding='utf-8',
+            stdout=PIPE,
+        )
+
+        for line in proc.stdout:
+            line_list = line.split(' ')
+            rel_path = os.path.join(*Path(line_list[0]).parts[-3:])
+            line_list[0] = rel_path
+            line = ' '.join(line_list)
+            yield line.strip()
+
+        # returncode 1 is app exception, 0 is no linter err, 2 is linter err
+        if proc.wait() not in (0, 2):
```

Comment:
Please do not hard code exit codes if they're anything except `0` or `-1`.

Reply:
Added a constant here to denote the meaning of the `ansible-lint` return code